codegen.mts

```diff
@@ -53,93 +45,99 @@ const config: CodegenConfig = {
           SchemaWarningConnection:
             '../modules/schema/module.graphql.mappers#SchemaWarningConnectionMapper',
           OrganizationConnection:
-            '../shared/mappers#OrganizationConnection as OrganizationConnectionMapper',
-          UserConnection: '../shared/mappers#UserConnection as UserConnectionMapper',
-          ActivityConnection: '../shared/mappers#ActivityConnection as ActivityConnectionMapper',
-          MemberConnection: '../shared/mappers#MemberConnection as MemberConnectionMapper',
-          ProjectConnection: '../shared/mappers#ProjectConnection as ProjectConnectionMapper',
-          TargetConnection: '../shared/mappers#TargetConnection as TargetConnectionMapper',
-          SchemaConnection: '../shared/mappers#SchemaConnection as SchemaConnectionMapper',
-          TokenConnection: '../shared/mappers#TokenConnection as TokenConnectionMapper',
+            '../modules/organization/module.graphql.mappers#OrganizationConnectionMapper',
+          UserConnection: '../modules/auth/module.graphql.mappers#UserConnectionMapper',
+          ActivityConnection: '../modules/activity/module.graphql.mappers#ActivityConnectionMapper',
+          MemberConnection: '../modules/auth/module.graphql.mappers#MemberConnectionMapper',
+          ProjectConnection: '../modules/project/module.graphql.mappers#ProjectConnectionMapper',
+          TargetConnection: '../modules/target/module.graphql.mappers#TargetConnectionMapper',
+          SchemaConnection: '../modules/schema/module.graphql.mappers#SchemaConnectionMapper',
+          TokenConnection: '../modules/token/module.graphql.mappers#TokenConnectionMapper',
           OperationStatsValuesConnection:
-            '../shared/mappers#OperationStatsValuesConnection as OperationStatsValuesConnectionMapper',
+            '../modules/operations/module.graphql.mappers#OperationStatsValuesConnectionMapper',
           ClientStatsValuesConnection:
-            '../shared/mappers#ClientStatsValuesConnection as ClientStatsValuesConnectionMapper',
+            '../modules/operations/module.graphql.mappers#ClientStatsValuesConnectionMapper',
           SchemaCoordinateStats:
-            '../shared/mappers#SchemaCoordinateStats as SchemaCoordinateStatsMapper',
-          ClientStats: '../shared/mappers#ClientStats as ClientStatsMapper',
-          OperationsStats: '../shared/mappers#OperationsStats as OperationsStatsMapper',
-          DurationValues: '../shared/mappers#DurationValues as DurationValuesMapper',
-          SchemaComparePayload:
-            '../shared/mappers#SchemaComparePayload as SchemaComparePayloadMapper',
-          SchemaCompareResult: '../shared/mappers#SchemaCompareResult as SchemaCompareResultMapper',
-          SchemaCompareError: '../shared/mappers#SchemaCompareError as SchemaCompareErrorMapper',
```

Comment:
I cannot find `SchemaComparePayload`, `SchemaCompareResult` and `SchemaCompareError` anywhere in the schema so I think it's safe to remove.